packages/esm-ward-app/src/ward-view/ward-view.resource.ts

```diff
@@ -25,19 +32,23 @@ export function filterBeds(admissionLocation: AdmissionLocationFetchResponse): B
 }
 
 //TODO: This implementation will change when the api is ready
-export function getWardMetrics(beds: Bed[]): WardMetrics {
+export function getWardMetrics(beds: Bed[], wardPatientGroup: WardPatientGroupDetails): WardMetrics {
   const bedMetrics = {
     patients: '--',
     freeBeds: '--',
     capacity: '--',
   };
-  if (beds.length == 0) return bedMetrics;
+  if (beds.length == 0 || !wardPatientGroup) return bedMetrics;
   const total = beds.length;
   const occupiedBeds = beds.filter((bed) => bed.status === 'OCCUPIED');
   const patients = occupiedBeds.length;
   const freeBeds = total - patients;
-  const capacity = total != 0 ? Math.trunc((patients / total) * 100) : 0;
-  return { patients: patients.toString(), freeBeds: freeBeds.toString(), capacity: capacity.toString() + '%' };
+  const capacity = total != 0 ? Math.trunc((wardPatientGroup.totalPatientsCount / total) * 100) : 0;
+  return {
+    patients: wardPatientGroup.totalPatientsCount.toString(),
+    freeBeds: freeBeds.toString(),
+    capacity: capacity.toString() + '%',
```

Comment:
This should be localized so that it appears correctly in e.g. Arabic, where the percent sign is affixed to the left side.

Reply:
Yeah sure @brandones , i will make the changes. Thanks.